src/views/bootablevolumes/list/BootableVolumesList.tsx

```diff
@@ -43,6 +44,9 @@ const BootableVolumesList: FC<BootableVolumesListProps> = ({ namespace }) => {
   const { createModal } = useModal();
   const { bootableVolumes, error, loaded } = useBootableVolumes(namespace);
 
+  const [canCreateDS, canCreatePVC] = useCanCreateBootableVolume(namespace);
+  const canCreate = canCreateDS || canCreatePVC;
```

Comment:
I'll create the object because i think it's more descriptive. Good idea. But this OR operation is something peculiar about this page and don't think we need that in other places.